packages/editor/src/components/blocks/input/InputWithBrowser.tsx

```diff
@@ -0,0 +1,41 @@
+import { Button, Dialog, DialogContent, DialogPortal, DialogTrigger, Fieldset, Input, InputGroup } from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { useState } from 'react';
+import { Browser, BrowserType } from '../../browser/Browser';
+import './Input.css';
+import type { InputFieldProps } from '../../editor/properties/fields/InputField';
+
+const supportedFields: { [value: string]: BrowserType[] } = {
+  value: [BrowserType.Attribute]
+};
+
+export const InputWithBrowser = ({ label, value, onChange }: InputFieldProps) => {
+  const [open, setOpen] = useState(false);
+  //const triggerStyle: React.CSSProperties = browserTypes ? { visibility: 'visible' } : { visibility: 'collapse' }; // TODO: Decide on method to decide
+  const triggerStyle: React.CSSProperties = Object.keys(supportedFields).includes(label)
+    ? { visibility: 'visible' }
+    : { visibility: 'collapse' };
+  return (
+    <Dialog open={open} onOpenChange={setOpen}>
+      <Fieldset label={label}>
+        <InputGroup>
+          <Input value={value} onChange={e => onChange(e.target.value)} />
+          <DialogTrigger asChild style={triggerStyle}>
+            <Button icon={IvyIcons.ListSearch} aria-label='Browser' />
+          </DialogTrigger>
+        </InputGroup>
+      </Fieldset>
+      <DialogPortal container={document.getElementById('properties')}>
```

Comment:
This does not seem to work when using the Dialog components from "ui-components".
I tried the Dialog-Components directly from radix, there it seems to work.

This issue might originate in the ui-components as the "DialogContent" also uses a "DialogPortal" but does not specify the "container".

Reply:
Yes this is true, currently the dialog does not expose this container attribute. Totally correct analysis 👍 

However if we want to go to the new Browser design it should not be necessary anymore I think. But I'm not sure if you are aware of the new design:
![Screenshot 2024-06-17 at 08 13 04](https://github.com/axonivy/form-editor-client/assets/42733123/6b0c2f0c-585d-4ff4-b914-5ee4763f0a4f)

So it is panned, that it is more a dialog beside the config panel.